website/source/partials/builder/vsphere/iso/_CDRomConfig-not-required.html.md

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
This is a generated file that is generated based on builder/vsphere/iso/step_add_cdrom.go
What you can do is use the same logic for generating docs based on builder/vsphere/iso/step_remove_cdrom.go, and then you can write the docs as comment above 
```
RemoveCdrom bool `mapstructure:"remove_cdrom"`
```